docs/content/guides/cell-features/merge-cells/merge-cells.md

```diff
@@ -66,6 +66,58 @@ To initialize Handsontable with predefined merged cells, provide merged cells de
 
 :::
 
+## Optimizing rendering of the wide/tall merged cells
+
+Sometimes, you might need to render a table with merged cells that are very wide or very tall. In such cases, the standard rendering mode may not be optimal. In these situations, we suggest enabling the virtual DOM for the merged cells.
+
+To enable the merged cells virtualization mode, enable the `virtualized` option:
+
+::: only-for javascript
+
+```js
+mergeCells: {
+  virtualized: true,
+  cells: [{ row: 1, col: 1, rowspan: 200, colspan: 2 }]
+}
+```
+
+:::
+
+::: only-for react
+
+```jsx
+mergeCells={{
+  virtualized: true,
+  cells: [{ row: 1, col: 1, rowspan: 200, colspan: 2 }]
+}}
+```
+
+:::
+
+The example below uses virtualized merged cells. It's also recommended to increase the buffer of rendered rows/columns to minimize the flickering effects.
```

Comment:
Maybe: In the example below, virtualized merged cells are enabled. Additionally, increasing the buffer size for rendered rows and columns can help reduce flickering effects during scrolling.

Reply:
Both, I'd say, sound the same .. I'd leave it as it is :)